src/debasmita/A19_StringProgram.java

```diff
@@ -0,0 +1,33 @@
+package debasmita;
+
+public class A19_StringProgram {
+	
+	 int uppercasecnt=0,lowercasecnt=0,totalcnt=0,digitcnt=0;
```

Comment:
if this variable is common for other methods then we should write it at the instance level.